### PR TITLE
PropertyBinding: Convert to es6 class

### PR DESCRIPTION
**Pull request by ianpurvis**
Supports #19986

### CODE REVIEW

**Comment by mrdoob** [comment]
Is there a way to remove these `Object.assign()`s?

**Comment by DefinitelyMaybe** [comment]
Oh, you're not into them? I thought that given that code base had them, you would have preferred them to stay, changing incrementally as appose to full on. cool 👍 

Sorry for #21294 then, that was part of what that was about.

**Comment by ianpurvis** [comment]
Sounds good, I'll go ahead and tweak this PR to remove the `Object.assign` calls.

**Comment by ianpurvis** [comment]
@mrdoob All set... 64150b7

File: src/animation/PropertyBinding.js [hunk 1]
```diff
@@ -184,9 +191,9 @@ Object.assign( PropertyBinding, {
 
 		return results;
 
-	},
+	}
 
-	findNode: function ( root, nodeName ) {
+	static findNode( root, nodeName ) {
```
Comment (Mugen87) [inline src/animation/PropertyBinding.js:196]: The below `if` statement contains `"` characters which should not be the case.

Not rebasing makes it more possible produce a regression.

Comment (ianpurvis) [reply src/animation/PropertyBinding.js:196]: @Mugen87 Sorry for the delay- this is now rebased with single quotes.

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
@mrdoob Should we give this a go? It seems it's the last file in `src/animations` which is not yet converted.

**Comment by mrdoob** [comment]
Lets do it 👍

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**